hyperstart.go

```diff
@@ -430,6 +512,18 @@ func (h *hyper) startOneContainer(pod Pod, c Container) error {
 		return err
 	}
 
+	//TODO : Enter mount namespace
+
+	// Handle container mounts
+	fsmap, err := h.bindMountContainerMounts(pod.id, c.id, c.config.Mounts)
+	if err != nil {
+		h.bindUnmountAllRootfs(pod)
+		return err
+	}
+
```

Comment:
I'd rather do it from a more generic place, to avoid code duplication across hypervisors. I know we only support one for now, but it's an architecture discipline we should follow.